pkg/pdp/service/contract/contract.go

```diff
@@ -0,0 +1,139 @@
+package contract
```

Comment:
A mechanical change that modifies the PDP service to operate on the contract using an interface so we can mock it during testing.